flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/PartitionRequestClientFactoryTest.java

```diff
@@ -131,8 +129,6 @@ public void testNettyClientConnectRetryFailure() throws Exception {
 
 			factory.createPartitionRequestClient(serverAndClient.getConnectionID(0));
```

Comment:
While the commit message is quite direct, it seems a bit clumsy.

How about `[refactor][tests][network] Cleanup PartitionRequestClientFactoryTest`?

Reply:
Do you mean 
`[tests][network][refactor] Fix IDE warnings`
or
`[tests][network][refactor] Extract and use NettyServerAndClient.getConnectionID`
?